eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -35,7 +35,7 @@ function Invoke-GitHubAPIPost {
     else {
       $warning = "{0} with Uri [ $apiURI ] did not fire request because of empty body." -f (Get-PSCallStack)[1].FunctionName
       LogWarning $warning
-      exit 0
+      return $null
```

Comment:
I'm ok with this but what prompted this? I'd still like us to fix the callers to not have us ever hit this empty body case.